council/council.html

```diff
@@ -227,16 +226,16 @@ <h3 id="convincing-the-council"><a href="#convincing-the-council">II. Convincing
   supported by all is always preferred. Nothing in this discussion precludes such an approach or forces an adversarial judicial proceeding.
   Nonetheless, there should be a clear expectation about the starting point for Council deliberations.</p>
 
-<p>Objectors should not be looking for the Council to come up with new arguments to support their point of view.
+<p>Objectors should not be looking at the Council to come up with new arguments to support their point of view.
```

Comment:
`for` seems better here, but native speakers should confirm

Reply:
look at is correct.